### DISCUSSION TITLE
[HELP] Permission error - macOS Monterey

### DISCUSSION THREAD

**Discussion by jailton-silvasp** [elrumo/macOS_Big_Sur_icons_replacements#1453] [General]
Good night people,

I'm new here on the forum, I looked for something about my question, but I didn't find it...it happens that I'm having trouble changing the mac's default icons (example: Calendar, Finder, Calculator, maps, etc). It returns that I don't have permission, I already disabled SIP, but it still returns that I don't have permission.

How can you change these icons?

**Comment by SoCuul** [comment]
Try to add yourself as an owner of the file
